js_modules/dagit/packages/core/src/assets/AssetMaterializationGraphs.tsx

```diff
@@ -22,28 +27,17 @@ export const AssetMaterializationGraphs: React.FC<{
   const graphDataByMetadataLabel = extractNumericData(reversed, props.xAxis);
   const graphLabels = Object.keys(graphDataByMetadataLabel).slice(0, 20).sort();
 
-  const [collapsedLabels, setCollapsedLabels] = useStateWithStorage(
-    'hidden-graphs',
-    validateHiddenGraphsState,
-  );
-
-  const toggleCollapsed = React.useCallback(
-    (label: string) => {
-      setCollapsedLabels((current = []) =>
-        current.includes(label) ? current.filter((c) => c !== label) : [...current, label],
-      );
-    },
-    [setCollapsedLabels],
-  );
+  if (process.env.NODE_ENV === 'test') {
+    return <span />; // chartjs and our useViewport hook don't play nicely with jest
+  }
```

Comment:
Could we have manual mocks for the stuff that breaks in Jest?

Reply:
Hmm I think we could, let me try in a follow-up today!